sunpy/map/tests/test_mapbase.py

```diff
@@ -328,8 +328,8 @@ def test_submap(generic_map):
     assert submap.meta['naxis2'] == height / 2.
 
     # Check data
-    assert (generic_map.data[height/2:height,
-                             width/2:width] == submap.data).all()
+    assert (generic_map.data[int(height/2):int(height),
+                             int(width/2):int(width)] == submap.data).all()
```

Comment:
here too, `height` should be an `int` then, we should not put `int` around it. `/` to `//` too?